src/ert/_c_wrappers/job_queue/workflow_job.py

```diff
@@ -137,56 +136,25 @@ def run(self, ert: "EnKFMain", arguments: List[Any], verbose: bool = False) -> A
         if self.internal and self.script is not None:
             script_obj = ErtScript.loadScriptFromFile(self.script)
             self.__script = script_obj(ert)
-            result = self.__script.initializeAndRun(
-                self.argumentTypes(), arguments, verbose=verbose
-            )
-
-        elif self.internal and self.script is None:
+        elif self.internal:
             self.__script = FunctionErtScript(
                 ert,
                 self.function,
                 self.argumentTypes(),
                 argument_count=len(arguments),
             )
-            result = self.__script.initializeAndRun(
```

Comment:
I believe that that there is no valid value for `self.function` any more, so either we can go and see in the `src/ert/clib.so` for any function matching the signature `void * f(void*, stringlist_type*)` and if there is no such function we simply remove this code and make it into an error message. If we do keep this code around we should log usage to try to understand whether it is used. Also in order to call `FunctionErtScript`, `self.function` has to be not `None` so that should be in the `if` statement.